modules/templates/SSF/config.py

```diff
@@ -539,6 +539,32 @@ def task_rheader(r):
             return rheader
 
         attr["rheader"] = task_rheader
+
+        standard_postp = s3.postp
+
+        def postp(r, output):
+            if callable(standard_postp):
+                output = standard_postp(r, output)
+
+            if auth.is_logged_in():
+                filter_url = URL(c="project", f="task", args=[])
+                filter_url += "?~.project_member.person_id__eq=%s"
+                filter_url = filter_url % auth.s3_logged_in_person()
+                if r.representation == "html":
+                    output["showadd_btn"] = CAT(A(T("Create Task"),
```

Comment:
Check if user has "create" permissions. This is already done before coming here and "Create Task" would've been added only if the user had permissions, so can you introspect it from the output ?